package.json

```diff
@@ -27,6 +33,13 @@
     "typescript-eslint": "^0.0.1-alpha.0"
   },
   "dependencies": {
-    "csvtojson": "^2.0.10"
+    "@hapi/joi": "^17.1.1",
```

Comment:
I learned that joi is moving back from @hapi org to just `joi`. Do we still need both `@hapi/joi` and `joi` in dependencies?

Reply:
Good to know, will revalidate that